includes/Register_Widget.php

```diff
@@ -26,12 +26,14 @@
 
 namespace Google\Web_Stories;
 
+use Google\Web_Stories\Infrastructure\Registerable;
+use Google\Web_Stories\Infrastructure\Service;
 use Google\Web_Stories\Widgets\Stories;
 
 /**
  * Class RegisterWidget
  */
-class Register_Widget extends Service_Base {
+class Register_Widget implements Service, Registerable {
```

Comment:
There is no need to delay these filters being registered. So Service base is not needed here.